.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "Tuya Local Dev",
 	"image": "mcr.microsoft.com/devcontainers/python:1-3.13",
-	"postCreateCommand": "pip3 install --user -r requirements.txt",
```

Comment:
For some reason dev dependencies were not installed here, I assume it was a bug.

Reply:
Probably. This file was an old contribution from another user, I do not use vscode myself. Probably requirements.txt is enough to get completion working, at least for some imports, but dev is required for unit testing and probably for pulling in the HA imports.